assets/css/admin/style.scss

```diff
@@ -93,12 +93,33 @@ body.post-type-download #screen-meta {
 	padding: 0 20px;
 }
 
+#edd-header-branding {
+	vertical-align: middle;
+}
+
 #edd-header img {
-	display: block;
 	max-width: 300px;
 	margin: 0 0 0 20px;
 }
 
+.edd-header-page-title-wrap {
+	margin-top: 7px;
+	font-size: 22px;
+	line-height: 28px;
+	vertical-align: top;
+	display: inline;
+	margin-right: auto;
+	padding-left: 7px;
+}
+
+.edd-header-seperator {
+	margin-top: -2px;
+}
+
+.edd-header-page-title {
+	font-weight: 400;
```

Comment:
This span is already 400; this rule seems redundant.

Reply:
Now that we've converted this to an H1, we need to set it or else it ends up being defaulted to 600, like the admin styles for H1.